packages/fxa-content-server/tests/functional/sign_up.js

```diff
@@ -272,6 +272,7 @@ registerSuite('signup here', {
     },
 
     'signup via product page and redirect after confirm': async function() {
+      /*
```

Comment:
Looks like this file might also have some WIP changes; please be sure to delete the commented-out chunks before landing :+1:

Reply:
Not sure we want to delete these - we still need the tests. It's just that the tests either need to be rewritten, or we need a stub Stripe API

https://jira.mozilla.com/browse/FXA-1242

Reply:
Ah right. Could we use the official [stripe mock](https://github.com/stripe/stripe-mock) for this sign up flow happy path test? It seems like it'd be useful, despite its [limitations](https://github.com/stripe/stripe-mock#current-state-of-development).

Reply:
We haven't found that mock to be useful at all so far, but could be worth looking into. Would like to defer that over to FXA-1242 though